libafl_bolts/src/cli.rs

```diff
@@ -34,14 +34,14 @@
 //! use std::env;
 //!
 //! // make sure to add `features = ["qemu_cli"]` to the `libafl` crate in `Cargo.toml`
-//! use libafl_qemu::Emulator;
+//! use libafl_qemu::Qemu;
```

Comment:
It's mainly to make less confusing the difference between the low-level and the high-level struct for QEMU. I also believe it will be less confusing for people to have `Qemu` being the struct with all the low-level calls to QEMU.
But maybe I'm wrong?